### PR TITLE
Onboarding wizards (addresses #3952 )

### LINKED ISSUE #3952: firstMile: Update docs post-release of new Python, Javascript, and GO Onboarding flow
After the release of the new Python, Javascript, and GO onboarding flow, all users in production will have a different landing page as well as the option to follow an onboarding setup wizard for either Python, Javascript, or GO. 

Docs teammates can see these new onboarding flows as well as the new landing page currently in Staging. For help navigating to the functionality in staging, please contact @hoorayimhelping . 

Anywhere that code snippets are used to help coach/onboard new users via user web docs will need to be updated to match the new onboarding code snippets in the UI.  (Python, Javascript, or GO)

This release is targeted to occur within the next 3 weeks, tentatively on April 27th 2022. 

FYI @kristinarobinson @kelseiv

### PR DESCRIPTION
**Pull request by noramullen1**
Closes #3952

### CODE REVIEW

File: content/influxdb/cloud/get-started.md [hunk 1]
```diff
@@ -13,10 +13,11 @@ influxdb/cloud/tags: [get-started, install]
 
 After you've [signed up for InfluxDB Cloud](/influxdb/cloud/sign-up/), you're ready to get started:
 
-1. Do one of the following:
+- Do one of the following:
+   - [Write and query data using the programming language of your choice](#write-and-query-data-using-the-programming-language-of-your-choice).
    - Add [sample data](#add-sample-data).
    - [Use your own data](/influxdb/cloud/write-data/) to explore InfluxDB Cloud.
-2. [Create a notebook](#create-a-notebook):
```
Comment (Luckette) [inline content/influxdb/cloud/get-started.md:19]: Creating a notebook isn't part of the onboarding flow.

Comment (noramullen1) [reply content/influxdb/cloud/get-started.md:19]: Hi @Luckette, this is our general getting started doc. The notebooks section wasn't added or changed as part of this update. Like the sections about community templates and writing tasks, it's simply an option we're introducing the reader to for starting to interact with their data. 

You can see the full live current doc in context here: https://docs.influxdata.com/influxdb/cloud/get-started/

Let me know if you have any questions!

**Review by lwandzura — APPROVED** [review]
LGTM

File: content/influxdb/cloud/get-started.md (hunk 1 above)
Comment (Luckette) [reply content/influxdb/cloud/get-started.md:19]: Oh ok got it, thanks for explaining!

**Review by Luckette — APPROVED** [review]

**Pull request merged**